src/gluonts/model/gp_forecaster/__init__.py

```diff
@@ -11,6 +11,15 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-from ._estimator import GaussianProcessEstimator
+import warnings
+
+from gluonts.mx.model.gp_forecaster._estimator import GaussianProcessEstimator
```

Comment:
I think in all these, it's not necessary to import from `_estimator`:

```suggestion
from gluonts.mx.model.gp_forecaster import GaussianProcessEstimator
```